build.sbt

```diff
@@ -119,4 +119,4 @@ releaseProcess := Seq[ReleaseStep](
 // Test
 parallelExecution in Test := false
 logBuffered in Test := false
-testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDF")
+testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-oDTF")
```

Comment:
I'm not familiar with scalatest arguments, but I just give quick read of [this](http://www.scalatest.org/user_guide/using_the_runner), and I think you're missing some values after these arguments.
Unless what I have seen has nothing to do with this, in which case just ignore it

Reply:
So the `T` flag which I added is to show an extra reminder of failed and cancelled tests at the end. I find it quite useful as I don't need to scroll through all of the tests to see the failures - I'll add a comment.

Reply:
Oh Nice, I'll use these in my projects too 😄